pkg/authorization/util/util.go

```diff
@@ -14,6 +14,8 @@ import (
 // It returns the modified SubjectAccessReview.
 func AddUserToSAR(user user.Info, sar *authorization.SubjectAccessReview) *authorization.SubjectAccessReview {
 	sar.Spec.User = user.GetName()
+	// reminiscent of the bad old days of C.  Copies copy the min number of elements of both source and dest
+	sar.Spec.Groups = make([]string, len(user.GetGroups()), len(user.GetGroups()))
```

Comment:
@deads2k ooh, I'm very sorry I got that wrong. Thanks. I wonder what the rationale was for that particular golang sharp edge.